src/common.jl

```diff
@@ -109,6 +108,17 @@ function sumsq_percol(a::AbstractMatrix{T}) where {T}
     return r
 end
 
+function norm_percol(a::AbstractMatrix{T}) where {T}
+    n = size(a, 2)
+    √T = typeof(sqrt(oneunit(T)))
+    r = Vector{√T}(undef, n)
+    for j in 1:n
```

Comment:
I didn't get the need of `norm_percol` here given that it's equivalent to `sqrt!(sumsq_percol(a))`

FYI

```diff
-    for j in 1:n
+    @simd for j in 1:n
```

This makes the op 2x faster in my machine. I think we could do the same to `sumsq_percol` as well.

Reply:
When `a` is a matrix of `Int`s, then `sumsq_percol(a)` is a vector of `Int`s, and then you can't compute the `sqrt` in-place, see #106. So, `norm_percol` allocates a vector of the required type to store the square root. I'll add the `@simd` macro.